src/server/zset_family_test.cc

```diff
@@ -637,4 +637,13 @@ TEST_F(ZSetFamilyTest, ZDiff) {
   EXPECT_THAT(resp.GetVec(), ElementsAre("two", "2", "three", "3", "four", "4"));
 }
 
+TEST_F(ZSetFamilyTest, GeoAdd) {
+  EXPECT_EQ(1, CheckedInt({"geoadd", "Sicily", "13.361389", "38.115556", "Palermo"}));
+  EXPECT_EQ(1, CheckedInt({"geoadd", "Sicily", "15.087269", "37.502669", "Catania"}));
+  EXPECT_EQ(0, CheckedInt({"geoadd", "Sicily", "15.087269", "37.502669", "Catania"}));
+  auto resp = Run({"geohash", "Sicily", "Palermo", "Catania"});
+  ASSERT_THAT(resp, ArrLen(2));
+  EXPECT_THAT(resp.GetVec(), ElementsAre("sqc8b49rny0", "sqdtr74hyu0"));
```

Comment:
Nit: you could replace these 3 statements with:
```cpp
EXPECT_THAT(Run({"geohash", "Sicily", "Palermo", "Catania"}), IsRespArray(ElementsAre("sqc8b49rny0", "sqdtr74hyu0"))));
```

Reply:
I did not find anywhere `IsRespArray` predicate

Reply:
Sorry, no `Is`, just `RespArray`, see many examples here: https://github.com/dragonflydb/dragonfly/blob/main/src/server/cluster/cluster_family_test.cc

Reply:
btw, for my defense the WHOLE test was generated by copilot, including geo name, lat long constants and hashes 😮

Reply:
and it passed as is !!!